Library/Homebrew/cli/args.rb

```diff
@@ -33,13 +33,16 @@ def freeze_remaining_args!(remaining_args)
       end
 
       def freeze_named_args!(named_args, cask_options:)
+        options = {
+          override_spec: (:head if self[:HEAD?]),
+          force_bottle:  (true if self[:force_bottle?]),
+          flags:         (flags_only unless flags_only.empty?),
```

Comment:
Could these be pulled into variables or `options[:override_spec]` calls outside? The inline `()` usage is a bit hard to follow and not what we typically do.